app/views/supermarkets/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Supermarket Name: <%= @supermarket.name %></h1>
+
+<section class="supermarket_info">
+<h3>Supermarket Location <%= @supermarket.location %></h3>
+</section>
+
+<section class="customers">
+<h3>Customer who have Shopped at this Location: </h3>
+<% @supermarket.items.each do |item| %>
```

Comment:
Here is where we could create a method in our `supermarket.rb` model that gets us a unique list of customers using AR. Then you won't need to do any nested iterating.